apps/pie-docs/src/_includes/home-page.njk

```diff
@@ -0,0 +1,30 @@
+{% set heroSrc = ['/assets/img/heroes/', heroImageSrc] | join | url %}
+
+<div class="c-hero" data-test-id="site_content">
+    <div class="c-heroBanner">
+        <div class="c-content-container">
+            <div class="c-heroBanner-heading">
+                <div class="c-heroBanner-heading-text">
+                    <h1 class="alpha--standout">
+                        {{ title }}
+                    </h1>
+                    <p class="c-content-header-description">
+                        {{ description }}
+                    </p>
+                    <a class='o-btn' href={{ heroCtaHref | url }}>{{ heroCtaTitle }}</a>
+                </div>
+                <div class="u-showAboveMid c-heroBanner-image">
+                    <img src={{ heroSrc }} alt="" />
```

Comment:
<img width="384" alt="Screenshot 2023-07-26 at 08 29 59" src="https://github.com/justeattakeaway/pie/assets/16766185/04057730-de0a-4fd6-b6d3-f596c3ee9db3">

Unsure about this but might be worth checking if `alt` is the same as `alt=""` when using decorative images

Reply:
yep it is the same https://html.spec.whatwg.org/multipage/syntax.html#attributes-2 👍